src/components/common/SelectTypes.tsx

```diff
@@ -0,0 +1,85 @@
+import { Stack } from '@mui/material';
+import Checkbox from '@mui/material/Checkbox';
+import FormControl from '@mui/material/FormControl';
+import InputLabel from '@mui/material/InputLabel';
+import ListItemText from '@mui/material/ListItemText';
+import MenuItem from '@mui/material/MenuItem';
+import OutlinedInput from '@mui/material/OutlinedInput';
+import Select, { SelectChangeEvent } from '@mui/material/Select';
```

Comment:
Any reason why you do not import everything from `@mui/material` ?

Reply:
I used auto import of vs code.